From 01d2ebb468015ebd05f283b85e3c1f7ffe04dd3e Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Mon, 2 Oct 2023 14:21:21 +0200
Subject: [PATCH] com.webos.notification.role.json.in: Fix permission issues

Fixes:

Oct 02 12:30:58 qemux86-64 ls-hubd[291]: [] [pmlog] ls-hubd LSHUB_NO_OUT_PERMS {"DEST_APP_ID":"com.webos.surfacemanager","SRC_APP_ID":"com.webos.notification","EXE":"/usr/sbin/notificationmgr","PID":661} "com.webos.notification" does not have sufficient outbound permissions to communicate with "com.webos.surfacemanager" (cmdline: /usr/sbin/notificationmgr)

Oct 02 12:31:02 qemux86-64 ls-hubd[291]: [] [pmlog] ls-hubd LSHUB_NO_OUT_PERMS {"DEST_APP_ID":"com.webos.app.notification-11458","SRC_APP_ID":"com.webos.notification","EXE":"/usr/sbin/notificationmgr","PID":661} "com.webos.notification" does not have sufficient outbound permissions to communicate with "com.webos.app.notification-11458" (cmdline: /usr/sbin/notificationmgr)

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending

 files/sysbus/com.webos.notification.role.json.in | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/files/sysbus/com.webos.notification.role.json.in b/files/sysbus/com.webos.notification.role.json.in
index b245c53..7ccc1aa 100644
--- a/files/sysbus/com.webos.notification.role.json.in
+++ b/files/sysbus/com.webos.notification.role.json.in
@@ -10,10 +10,16 @@
                 "com.palm.bus",
                 "com.webos.settingsservice",
                 "com.palm.db",
+                "com.webos.service.eventmonitor",
                 "com.webos.service.tv.systemproperty",
                 "com.palm.systemservice",
                 "com.webos.applicationManager",
-                "com.webos.appinstaller"
+                "com.webos.appinstaller",
+                "com.webos.surfacemanager*",
+                "com.webos.app.notification*",
+                "com.webos.monitor*",
+                "com.palm.webappmanager",
+                "com.webos.lunasend-*"
             ]
         }
     ]
